product_form_account_move_link/models/product_product.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    account_move_lines_count = fields.Integer(
+        compute="_compute_account_move_lines_count", string="Move Lines"
+    )
+
+    def _compute_account_move_lines_count(self):
+        if not self.user_has_groups("account.group_account_user") or not self.ids:
```

Comment:
```suggestion
        if not self.user_has_groups("account.group_account_invoice") or not self.ids:
```

I would suggest to use the group_account_invoice which is the base one for accounting